pyqtgraph/graphicsItems/PlotDataItem.py

```diff
@@ -530,7 +530,7 @@ def getData(self):
             if self.opts['logMode'][0]:
                 x = np.log10(x)
             if self.opts['logMode'][1]:
-                y = np.log10(y)
+                y = np.log10(1.0+y)
             #if any(self.opts['logMode']):  ## re-check for NANs after log
                 #nanMask = np.isinf(x) | np.isinf(y) | np.isnan(x) | np.isnan(y)
                 #if any(nanMask):
```

Comment:
it looks like this code block that is now commented out was meant to deal with this problem. @campagnola, what was the rationale for removing this code block?